docs/standard/garbage-collection/latency.md

```diff
@@ -7,42 +7,50 @@ helpviewer_keywords:
   - "garbage collection, latency modes"
 ms.assetid: 96278bb7-6eab-4612-8594-ceebfc887d81
 ---
-# Latency Modes
-To reclaim objects, the garbage collector must stop all the executing threads in an application. In some situations, such as when an application retrieves data or displays content, a full garbage collection can occur at a critical time and impede performance. You can adjust the intrusiveness of the garbage collector by setting the <xref:System.Runtime.GCSettings.LatencyMode%2A?displayProperty=nameWithType> property to one of the <xref:System.Runtime.GCLatencyMode?displayProperty=nameWithType> values.
+# Latency modes
 
- Latency refers to the time that the garbage collector intrudes in your application. During low latency periods, the garbage collector is more conservative and less intrusive in reclaiming objects. The <xref:System.Runtime.GCLatencyMode?displayProperty=nameWithType> enumeration provides two low latency settings:
+To reclaim objects, the garbage collector (GC) must stop all the executing threads in an application. *Latency* refers to the time that the garbage collector intrudes in your application. In some situations, such as when an application retrieves data or displays content, a full garbage collection can occur at a critical time and impede performance. You can adjust the intrusiveness of the garbage collector by setting the <xref:System.Runtime.GCSettings.LatencyMode%2A?displayProperty=nameWithType> property to one of the <xref:System.Runtime.GCLatencyMode?displayProperty=nameWithType> values.
```

Comment:
The "Latency refers to ..." sentence seems vague to me. I think your definition of "Low latency" under the following H2 is clearer and makes this sentence unnecessary.